glide-core/Cargo.toml

```diff
@@ -58,12 +59,13 @@ standalone_heartbeat = []
 rsevents = "0.3.1"
 socket2 = "^0.5"
 tempfile = "3.3.0"
-rstest = "^0.23"
+rstest = "^0.25"
 serial_test = "3"
-criterion = { version = "^0.5", features = ["html_reports", "async_tokio"] }
-which = "6"
-ctor = "0.2.2"
+criterion = { version = "^0.6", features = ["html_reports", "async_tokio"] }
+which = "7"
+ctor = "0.4"
 redis = { path = "./redis-rs/redis", features = ["tls-rustls-insecure"] }
+rustls = { version = "0.23", features = ["ring"], default-features = false}
```

Comment:
Why do you add it here? 
Aren't it being used only in redis-rs, where it is already stated as dependency?